NutriApp/Views/About/AboutPage.xaml.cs

```diff
@@ -1,6 +1,8 @@
+using NutriApp.Components.ContentPageCustomer;
+
 namespace NutriApp.Views.About;
 
-public partial class AboutPage : ContentPage
+public partial class AboutPage : BaseContentPage
```

Comment:
```suggestion
public partial class AboutPage
```

Nem precisa usar aqui, o IDE até fala que é redundante, não?